### PR TITLE
docs(sdk): issuance ack API

### PR DESCRIPTION
**Pull request by rolsonquadras**

### CODE REVIEW

**Review by talwinder50 — APPROVED** [review]

File: cmd/wallet-sdk-gomobile/docs/usage.md [hunk 1]
```diff
@@ -789,9 +789,17 @@ val credentials = interaction.requestCredentialWithPreAuth(didDocument.assertion
 
 val issuerURI = interaction.issuerURI() // Optional (but useful)
 
-interaction.requireAcknowledgment() // return true if the issuer requires acknowledgment that credentials are accepted or rejected by the user.
-interaction.acknowledgeSuccess() // acknowledgment issuer that the user accepted credentials.
-interaction.acknowledgeReject() // acknowledgment issuer that the user rejected credentials.
+// Issuance Acknowledgment
+interaction.requireAcknowledgment() // returns true, if the issuer requires acknowledgment that credentials are accepted or rejected by the user.
+interaction.Acknowledgment() // get the Acknowledgment object, if issuer requires ack.
```
Comment (vkubiv) [inline cmd/wallet-sdk-gomobile/docs/usage.md:794]: Acknowledgment and Serialize functions should starts lowercase

File: cmd/wallet-sdk-gomobile/docs/usage.md [hunk 2]
```diff
@@ -789,9 +789,17 @@ val credentials = interaction.requestCredentialWithPreAuth(didDocument.assertion
...
+
+// use this API to get the opaque string, which needs to be submitted to the Acknowledgment APIs
+val ackTkn = interaction.Acknowledgment().Serialize()
+
+val ack = NewAcknowledgment(ackTkn)
```
Comment (vkubiv) [inline cmd/wallet-sdk-gomobile/docs/usage.md:799]: NewAcknowledgment ->  Acknowledgment

Comment (rolsonquadras) [reply cmd/wallet-sdk-gomobile/docs/usage.md]: Updated

File: cmd/wallet-sdk-gomobile/docs/usage.md (hunk 1 above)
Comment (rolsonquadras) [reply cmd/wallet-sdk-gomobile/docs/usage.md]: Updated

**Review by vkubiv — APPROVED** [review]

**Pull request merged**